create_database.py

```diff
@@ -0,0 +1,107 @@
+"""Read in a 'user data' text file and a 'format style' text file.
+Write the files to an SQLite database file for storage.
+"""
+import sqlite3
+import date_validation
+from parse import parse
+
+def read_file(file_name):
+    """Read in data from a file and return it."""
+    with open(file_name, 'r') as some_file:
+        data = some_file.read()
+    return data
+
+def chunk_data(file_data, size):
+    """Take in data as a string and an integer as a string. Return a
+    list containing chunks of data that are 'size' number of lines
+    each.
+    """
+    data_lines = file_data.splitlines()
+    assert data_lines, "There is no data!"
+    assert int(size), "An integer in the form of a string must be provided."
+    assert 0 < int(size) <= len(data_lines), "A valid size must be provided."
+    assert len(data_lines) % size == 0, "Data is incomplete."
+    return [data_lines[pos:pos + size] for pos in range(0, len(data_lines), size)]
+
+def create_user_table(db_filename, users_filename):
```

Comment:
Function is needlessly coupling file reading with database opening.

Why not have the function take in an active database object and an active file rather than opening them both.